src/doc/man/cargo-bench.md

```diff
@@ -28,10 +28,11 @@ similarly named benchmarks like `foobar`):
 
     cargo bench -- foo --exact
 
-Benchmarks are built with the `--test` option to `rustc` which creates an
-executable with a `main` function that automatically runs all functions
-annotated with the `#[bench]` attribute. Cargo passes the `--bench` flag to
-the test harness to tell it to run only benchmarks.
+Benchmarks are built with the `--test` option to `rustc` which creates a
+special executable by linking your code with libtest. The executable
+automatically runs all functions annotated with the `#[test]` attribute.
```

Comment:
```suggestion
automatically runs all functions annotated with `#[test]` and `#[bench]` attributes.
```

I was slightly confused reading this due to the mention of `#[test]`.  I'm not sure if this is any clearer. This is delving into some low-level details that is mostly not relevant to the average user.

Reply:
Ok. Now I feel like mentioning `#[bench]` is enough. I'll put it back then.